api/tests/unit/domain/models/Scorecard_test.js

```diff
@@ -114,6 +114,22 @@ describe('Unit | Domain | Models | Scorecard', () => {
       });
     });
 
+    context('when the competence evaluation has been reset and some knowledgeElements exist', () => {
```

Comment:
peut-être spécifier l'autre contexte en `when the competence evaluation has been reset but no knowledgeElement exist` au lieu de `when the competence evaluation has been reset`